### ISSUE TITLE
Error: An operation has `None` for gradient when Tensorboard callback

### ISSUE THREAD

**Issue by btekgit** [keras-team/keras#13090]
**System information**  
- Have I written custom code (as opposed to using example directory):  
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):  
- TensorFlow backend: yes  
- TensorFlow version:  v1.14.0-rc1-22-gaf24dc9 1.14.0
- Keras version:  2.2.4
- Python version:  3.7.3
- CUDA/cuDNN version:  10.0
- GPU model and memory:  GTX 1070
